src/components/file-input/examples/file-input-type-filtering.tsx

```diff
@@ -0,0 +1,81 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Example of a file input component with type filtering
+ */
+@Component({
+    tag: 'limel-example-file-input-type-filtering',
+    shadow: true,
+})
+export class FileInputTypeFilteringExample {
+    @State()
+    private disabled = false;
+
+    @State()
+    private readonly = false;
+
+    @State()
+    private multiple = false;
+
+    @State()
+    private files: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-input
+                onFilesSelected={this.handleFilesSelected}
+                accept="image/*"
+                disabled={this.disabled || this.readonly}
+                multiple={this.multiple}
+            >
+                <limel-button label="Select an image" />
+                {this.files.map((file) => (
```

Comment:
Should we have the chips inside `limel-file-input`. I personally think it's a bit confusing that when you click a chip it opens the file selection dialog. 😅

Reply:
It would be nice in this example to be able to remove the files as well, to show how it's done.

Reply:
I agree. 😅
See https://github.com/Lundalogik/lime-elements/pull/2819/commits/d3850a937286572606536aa31354839dec8b30dd